doc/guides/deploy-without-root.rst

```diff
@@ -0,0 +1,92 @@
+Deploying without Root
+===
+
+Requirements
+---
+
+NixOps 2.0 allows for deploying as users other than root, as long
+as the deploying user meets two requirements:
+
+1. The user can become root without needing to type a password.
+2. Nix considers the user to be a `"trusted user" <https://nixos.org/nix/manual/#conf-trusted-users>`_.
+
+In this guide, we will use passwordless sudo.
+
+This guide assumes:
```

Comment:
"In this guide... This guide assumes" is repetitive. Maybe something like:

```suggestion

Our assumptions include:
```

or

```suggestion

We assume:
```

(newline because all the other lists do the same)